src/providers/cryptoauthlib/key_management.rs

```diff
@@ -30,11 +30,17 @@ impl Provider {
             Type::Aes => Ok(rust_cryptoauthlib::KeyType::Aes),
             Type::EccKeyPair {
                 curve_family: EccFamily::SecpR1,
+            } => {
+                if attributes.bits == 256 || attributes.bits == 0 {
+                    Ok(rust_cryptoauthlib::KeyType::P256EccKey)
+                } else {
+                    Err(ResponseStatus::PsaErrorNotSupported)
+                }
             }
-            | Type::EccPublicKey {
+            Type::EccPublicKey {
                 curve_family: EccFamily::SecpR1,
             } => {
-                if attributes.bits == 256 {
+                if attributes.bits == 512 || attributes.bits == 0 {
```

Comment:
I have an impression neither value is actually that important. From the interface point of view I would recommend to keep 0 as "do not care at the moment" just like elsewhere and 256 as the hard requirement of the provider.

Reply:
From the Parsec API point of view the two values have to be accepted so that the operation is implemented correctly on the crypto auth lib provider. That is so all providers supporting the same operation share the same behaviour.

Reply:
If I understand correctly this `attributes.bits == 256 || attributes.bits == 0` is the correct expression.

Reply:
Yes that's correct!